packages/core/cms/faststore/content-types.json

```diff
@@ -271,6 +271,46 @@
                       "default": "Receive Tomorrow"
                     }
                   }
+                },
+                "tags": {
+                  "title": "Delivery Promise tags",
+                  "type": "object",
+                  "description": "The tag specified in this section will be displayed in product cards. Only one tag can be displayed.",
+                  "properties": {
+                    "enabled": {
```

Comment:
question: Since we can only have one of them displayed, do you think we can omit this prop and leave just `deliveryOptionTag.enabled` and `dynamicEstimateTag.enabled`? Or this might change in the future?